docs/adr/0032-fix-second-account-persistence-race-condition.md

```diff
@@ -0,0 +1,100 @@
+# 32. Fix Second Account Persistence Race Condition and Database Mismatch
+
+Date: 2025-12-10
+
+## Status
+
+Accepted
+
+## Context
```

Comment:
should add: when we change db from hive to sqfite alongsite with using upstream matrix sdk